src/rshim_pcie.c

```diff
@@ -794,6 +820,7 @@ static void rshim_pcie_intr(rshim_pcie_t *dev)
   switch (info.rst_state) {
   case RSHIM_PCIE_RST_STATE_REQUEST:
     RSHIM_INFO("NIC reset ACK\n");
+    info.pcie = 1;
```

Comment:
`info.pcie = 1` is in all three defined cases here. If the intent is to run always, including the `default` case, then we can move it outside of the `switch`.